doc/quick-start-user-guide.md

```diff
@@ -68,30 +60,65 @@ clientaddress = "192.168.56.101:24007"
 etcdcurls = "http://192.168.56.101:2379"
 etcdpurls = "http://192.168.56.101:2380"
 ```
-
 Replace the IP address accordingly on each node.
 
-**Start glusterd2 process:** Glusterd2 is not a daemon and currently can run only in the foreground.
+### Using external etcd instead of embedded etcd.
+
+By default glusterd2 uses embedded etcd, but glusterd2 also supports usage of external etcd.
```

Comment:
The tone of this section must reflect that external etcd is the default and is the recommended way and supported way. It currently says otherwise.